packages/www/Dockerfile.dockerignore

```diff
@@ -0,0 +1,26 @@
+# Package to build
+packages/*
+!packages/api
+!packages/design-system
+!packages/www
+
+# General git and node frameworks files
+.git
+.github
+.cache
+**/node_modules
+**/coverage
+**/yarn-error.log
+**/.next
+
+# API-specific
+*/*/minio
+*/*/dist
+*/*/docs
+
+# WWW-specific
+*/*/.next
+*/*/out
```

Comment:
I'm wondering if we should have only 1 `.dockerignore` in the repo root instead 🤔 
Like feels there are some stuff copied from the API docker ignore, and maybe some stuff that could be copied back well. In the end might be simpler to just have them be the same.

If it is in the root, it can have the regular `.dockerignore` name and it'd be used for all docker builds since it would be in the context root (or at least that's what they say 😆 )

Reply:
The main difference being that `api` can ignore all of the other packages, whereas www needs them as dependencies... kinda nice to have the api build a bit faster and leaner.

Reply:
Oh that's true, forgot about that part